RDF-Pipeline/t/tests/0021_Serialize_on_HEAD_request_Issue_20/test-script

```diff
@@ -22,8 +22,17 @@ $ENV{RDF_PIPELINE_WWW_DIR} = $wwwDir;
 ############ CUSTOMIZE HERE ################
 
 # This test verifies that serState is not refreshed on a HEAD request.
-my $serState = "$wwwDir/cache/aba/serState";
-my $serStateLM = "$wwwDir/lm/FILE/%2Fhome%2Fdbooth%2Frdf-pipeline%2FPrivate%2Fwww%2Fcache%2Faba%2FserState";
+### The serState file path used to look like this:
+# my $serState = "$wwwDir/cache/aba/serState";
+### But now looks like this:
+# /var/www/cache/URI/SHORT_aba_HASHhQy3E2e4CUWxhwDVUNSYXgA/serState
```

Comment:
Why not $wwwDir/cache/URI/SHORT_aba_HASHhQy3E2e4CUWxhwDVUNSYXgA/serState ?   Do we really have it hard coded to //var/www?